Project.toml

```diff
@@ -8,6 +8,7 @@ Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MKL_jll = "856f044c-d86e-5d09-b602-aeab76dc8ba7"
 PackageCompiler = "9b87118b-4619-50d2-8e1e-99f35a4d4d9d"
+Preferences = "21216c6a-2e73-6563-6e65-726566657250"
```

Comment:
Is Preferences.jl 1.6 only? If so you might need to bump the julia required version as well.

Reply:
Ah yeah, that's missing on my todo list.

Question is: do we want to drop support for Julia versions < 1.6 entirely or, for example, set a fixed `mkl_provider = mkl_jll` along the lines of https://github.com/JuliaPackaging/Preferences.jl#conditional-loading ?

I'd opt for the former (dropping support) since 1.6 will likely become the next LTS soon anyways.